tests/storage/linstor/conftest.py

```diff
@@ -12,57 +12,85 @@
 LINSTOR_PACKAGE = 'xcp-ng-linstor'
 
 @pytest.fixture(scope='package')
-def lvm_disk(host, sr_disk_for_all_hosts):
-    device = '/dev/' + sr_disk_for_all_hosts
+def lvm_disk(host, sr_disks_for_all_hosts, provisioning_type):
```

Comment:
Maybe `lvm_disks` if you change the logic here. :)

Reply:
Yes :)